build-tools/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM amd64/ubuntu:bionic as builder
```

Comment:
Why not the official rust image: https://hub.docker.com/_/rust ?

Reply:
good catch, modified